src/test/java/org/phoebus/channelfinder/AAChannelProcessorTest.java

```diff
@@ -0,0 +1,55 @@
+package org.phoebus.channelfinder;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.phoebus.channelfinder.processors.AAChannelProcessor.ArchivePV;
+
+public class AAChannelProcessorTest {
+
+    @Test
+    public void archivePropertyParsing() {
+        ArchivePV ar = new ArchivePV();
+        ar.setPv("sim://testing");
+        ar.setSamplingParameters("monitor@1.0");
+
+        assertEquals(ar.getPv(), "sim://testing");
+        assertEquals(ar.getSamplingmethod(), "MONITOR");
+        assertEquals(ar.getSamplingperiod(), Float.valueOf("1.0"));
+
+        ar.setSamplingParameters("scan@.01");
+        assertEquals(ar.getSamplingmethod(), "SCAN");
+        assertEquals(ar.getSamplingperiod(), Float.valueOf(".01"));
```

Comment:
Thinking of some other cases to test.  Some which I think should succeed:

```
MONITOR@1
MONITOR@0.01
MONITOR@1 ignore me
MONITOR@0.01 ignore me
```

And some which I think should fail (somehow):

```
INVALID
oops@
@1.0
@blah
INVALID@1
SCAN @1
SCAN@ 1.0
SCAN@1invalid
SCAN@invalid
MONTOR@1
```